apps/settings/elements/homescreen.html

```diff
@@ -13,12 +13,6 @@ <h2 data-l10n-id="vertical-homescreen-layout">Layout</h2>
         <ul>
           <li>
             <p data-l10n-id="vertical-homescreen-icon-layout">Icon Layout</p>
-            <div class="button icon icon-dialog">
-              <select name="grid.layout.cols">
-                <option value="3" data-l10n-id="vertical-homescreen-three-columns">Three Columns</option>
-                <option value="4" data-l10n-id="vertical-homescreen-four-columns">Four Columns</option>
-              </select>
-            </div>
```

Comment:
Just removing this one select element leave us with an empty panel, doesn't it? I think you should probably remove this entire file and the references to it, right?